### ISSUE TITLE
Version mismatch in link to jshint docs

### ISSUE THREAD

**Issue by deg** [gruntjs/grunt-contrib-jshint#234]
For [good reason](https://github.com/gruntjs/grunt-contrib-jshint/issues/230#issuecomment-165207452) this does not yet support the latest jshint v2.9+.

But, the [readme](https://github.com/gruntjs/grunt-contrib-jshint/blob/master/README.md) references the  latest version of the [JSHint documentation](http://www.jshint.com/docs/) which led me to surprise and debugging when "esversion:", etc. did not work for me.

Ideally, it would be great if readme.MD pointed to the JSHint docs for v2.8. But (based on my quick look) those docs may not be available. They don't seem to be in the github repo.  So, maybe the easiest solution is just to add a warning line to the readme file.

**Comment by nicolo-ribaudo** [comment]
Related - jshint/jshint.github.io#29

**Comment by dgisser** [comment]
Wasn't this fixed in 3b3fed2ff204db439aeabdf1c11535b2310fee64? Can't we close this?

**Comment by estebanz01** [comment]
I confirm that with latest contrib-jshint version (2.1.0 at the moment), we have to specify `esnext` for ES6 linting, because it triggers a conflict error if `esversion` is used:

```
 Incompatible values for the 'esversion' and 'esnext' linting options. (0% scanned).
```
